sources/packages/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -246,6 +259,86 @@ export class InstitutionLocationInstitutionsController extends BaseController {
       applicationOfferingName: offering.name,
       applicationProgramDescription: offering.educationProgram.description,
       applicationProgramName: offering.educationProgram.name,
+      applicationProgramCredential: credentialTypeToDisplay(
+        offering.educationProgram.credentialType,
+      ),
+      applicationProgramDelivery: deliveryMethod(
+        offering.educationProgram.deliveredOnline,
+        offering.educationProgram.deliveredOnSite,
+      ),
+      applicationOfferingStudyDelivery: offering.offeringDelivered,
+      applicationOfferingStudyBreak: offering.studyBreaks.map((studyBreak) => ({
+        breakStartDate: dateString(studyBreak.breakStartDate),
+        breakEndDate: dateString(studyBreak.breakEndDate),
+      })),
+      applicationOfferingTuition: offering.actualTuitionCosts,
+      applicationOfferingProgramRelatedCosts: offering.programRelatedCosts,
+      applicationOfferingMandatoryFess: offering.mandatoryFees,
+      applicationOfferingExceptionalExpenses: offering.exceptionalExpenses,
     };
   }
+
+  /**
+   * Save scholastic standing and create new assessment.
+   * @param _locationId location id to check whether the
+   * request user has the permission to this location.
+   * @param applicationId application id.
+   * @param payload Scholastic Standing payload.
+   */
+  @ApiBadRequestResponse({ description: "Invalid form data" })
+  @HasLocationAccess("locationId")
+  @Post(":locationId/application/:applicationId/scholastic-standing")
+  async saveScholasticStanding(
+    @Param("locationId") _locationId: number,
```

Comment:
If the location is not used in the controller body it is a smell that there is some authorization missing. For this case, we need to ensure that the location has access to the application. Please do the check using the `locationId`.